.travis.yml

```diff
@@ -1,17 +1,14 @@
 sudo: false
 language: php
 php:
-- '5.6'
-- '7.0'
 - '7.1'
 - '7.2'
```

Comment:
Make sure you get it running on at least PHP 7.2 first, but we should also add PHP 7.3 and (maybe) PHP 7.4 to this list.

PHP 7.1 may or may not work; if it doesn't, that's okay, because it's EOL in December anyway.